projects/distributed-tracing/src/shared/components/table/data-cell/metric/metric-table-cell-renderer.component.ts

```diff
@@ -1,14 +1,14 @@
 import { ChangeDetectionStrategy, Component, Inject, Optional } from '@angular/core';
 import { Dictionary, FormatterStyle } from '@hypertrace/common';
 import {
-  TABLE_CELL_RENDERER_CELL_DATA,
-  TABLE_CELL_RENDERER_COLUMN_CONFIG,
-  TABLE_CELL_RENDERER_COLUMN_INDEX,
-  TABLE_CELL_RENDERER_ROW_DATA,
   TableCellAlignmentType,
   TableCellRenderer,
   TableCellRendererComponent,
-  TableColumnConfig
+  TableColumnConfig,
```

Comment:
Why are these ordering changes showing up? Any changes in the default lint rule ?

Reply:
prettier and tslint have disagreed on the alphabetical ordering wrt caps - I think that finally got resolved in https://github.com/palantir/tslint/pull/4064